speculos/api/events.py

```diff
@@ -31,13 +31,18 @@ def remove_client(self, client):
         self.logger.debug("events: client exited")
         self.clients.remove(client)
 
+    def clear_events(self):
+        self.logger.info("Clearing events")
+        self.screen_content = []
+
     def broadcast(self, event: TextEvent):
-        self.logger.debug(f"events: broadcasting {asdict(event)} to ({len(self.clients)}) client(s)")
         if self.screen_content:
             # Reset screen content if new event is not below the last text line of
             # current screen. Event Y coordinate starts at the top of the screen.
             if event.y <= self.screen_content[-1].y + MIN_LINES_HEIGHT_DISTANCE:
-                self.screen_content = []
+                self.clear_events()
+
+        self.logger.info(f"events: broadcasting {asdict(event)} to ({len(self.clients)}) client(s)")
```

Comment:
Sounds more like `DEBUG` log than `INFO` to me.

+ avoid using f-strings with `logging` package, as it [lazily interpolates string whether it needs to or not](https://docs.python.org/3/howto/logging.html#optimization) (for instance, if it only outputs ERROR, it won't process DEBUG strings). It's cheap optimization, and can be significant with frequently-used functions as `logging` ones.

Reply:
Well it's interesting information that I feel everyone could be interested about when watching the test run
I honestly feel like ragger and speculos are lacking a lot of logs, even logs that are considered debug today
this verbosity is only show if either the test fails or if we explicitly ask `-s`, so I don't feel bad at all about adding logs

Reply:
Done